packages/api/app/index.js

```diff
@@ -13,10 +13,38 @@ const awsServerlessExpressMiddleware = require('aws-serverless-express/middlewar
 
 const { services } = require('@cumulus/aws-client');
 const { MissingRequiredEnvVar } = require('@cumulus/errors');
+const { secretsManager } = require('@cumulus/aws-client/services');
+const Logger = require('@cumulus/logger');
 
 const router = require('./routes');
 const { jsonBodyParser } = require('./middleware');
 
+const log = new Logger({ sender: '@api/index' });
+
+// Load Environment Variables
+// This should be done outside of the handler to minimize Secrets Manager calls.
+const initEnvVarsFunction = async () => {
+  if (!process.env.api_config_secret_id) {
```

Comment:
Inclined to use our `getRequiredEnvVar` from `@cumulus/common/env` here

Reply:
That's fair.    I tend to not want to use common and that's not a good inclination. 

738bdd622

Reply:
738bdd622

Reply:
738bdd622